tests/fuzz/fuzz_common.h

```diff
@@ -0,0 +1,62 @@
+//
+// Created by profanter on 18.07.17.
+// Copyright (c) 2017 fortiss GmbH. All rights reserved.
+//
+
+#ifndef OPEN62541_FUZZ_COMMON_H_H
```

Comment:
I duplicated it, because the testing_networklayer is split into c and h files which makes the build for fuzz more complex.

Can we change the testing_networklayer to only have one header file?